rfc/0014-attribute-based-valuetypes/README.md

```diff
@@ -42,47 +37,61 @@ valuetype CorrelationCoefficient oftype decimal {
 }
 ```
 
-This syntax is smart in that you don't have to list and name an attribute but rather rely on an implicit 'value' attribute. Still, I propose to make that attribute explicit. New syntax would be:
+This syntax is smart in that you don't have to list and name an attribute but rather rely on an implicit 'value' attribute.
+
+The current syntax, in which an underlying value type is referenced through 'oftype', it does not use instantiation/inheritance but rather composition.
+The purpose of using composition is that it is
+
+1. more similar to traditional approaches and
+2. is needed anyway for multi-attribute value types.
+
+Thus, this RFC removes the `oftype` keyword in a value type definitions and proposes an attribute-based syntax that is extendable to a future multi-attribute syntax.
+
+## Explanation
+
+I propose to make that attribute explicit. The new syntax would be:
 
 ```jayvee
 valuetype CorrelationCoefficient {
-  attributes: [
-    value oftype decimal;
-  ];
-  constraints: [
-    MinusOneToPlusOneRange; // Don't know how to attach this to value attribute
-  ];
+  property correlation oftype decimal;
+
+  constraint minusOneToPlusOneRange: MinusOneToPlusOneRange(value = correlation);
 }
+
+constraint MinusOneToPlusOneRange on decimal:
+  value >= -1 and value <= 1;
 ```
 
-While more verbose, it prepares the way for 
+The syntax is kept flat rather than using arrays.
```

Comment:
```suggestion
The syntax is kept flat rather than using arrays in alignment with the [Jayvee Design Principles](https://jvalue.github.io/jayvee/docs/0.4.0/dev/design-principles#jayvee-manifesto).
```

Reply:
This sentence does not make sense anymore if we apply @rhazn's suggestion [above](https://github.com/jvalue/jayvee/pull/595/files#r1650751505).